docs/articles/getting-started/simple-example.md

```diff
@@ -1,85 +1,117 @@
-# Getting Started
+## A Simple Example
 
-## Baby Steps
+Let's write our first Bytewax Dataflow. Be sure that you've followed the instructions
+for installing bytewax in [Installing bytewax](/docs/articles/getting-started/installation.md).
```

Comment:
This link format is correct when navigating through our GH repository, but it'd result in an error on our website (where the path is: `/docs/getting-started/installation/`) if we merge it now. 

If we want to keep it that way in the source files, I need to check if there's an option to override link hrefs while parsing markdown in the website app

Reply:
FYI: I prepared a branch in our website repo with a fix that replaces the "internal GH repo" link format (e.g. /docs/articles/[category]/[filename].md) to the format we're using in the app (e.g. /docs/[category]/[filename]) during the markdown parsing step - so keeping that format in the source shouldn't result in any errors on front-end. 

Let me know if we want to use it, then I'll deploy the branch on staging for testing